src/modules/reading-room/components/KeywordsFilterForm/KeywordsFilterForm.tsx

```diff
@@ -102,6 +97,10 @@ const KeywordsFilterForm: FC<KeywordsFilterFormProps> = ({ children, className }
 
 	const onKeyDown = (e: KeyboardEvent<HTMLInputElement>) => onKey(e, [...keysEnter], saveInput);
 
+	const components = {
+		...TAGS_INPUT_COMPONENTS,
+		DropdownIndicator: () => <div className="u-pr-8" />,
+	};
```

Comment:
Didn't i move this one ouside the component after a PR remark from you?
Why are you moving it back inside?

Reply:
Didn't even open this file, presumably github glitching out